alfasim_sdk/hook_specs_gui.py

```diff
@@ -17,3 +17,31 @@ def alfasim_get_additional_variables():
     This variable can be used to store internal data from the plugin,
     or it can be used to expose data to the user in the plot window.
     """
+
+
+@hookspec
+def alfasim_get_phase_properties_calculated_from_plugin():
+    """
+    Must return a list of phases in which state variables will be computed for. In order to
+    implement the properties, HOOK_CALCULATE_STATE_VARIABLE must be implemented on the C plugin.
+
+    Example:
+    from alfasim_sdk.constants import GAS_PHASE
```

Comment:
I think that an user defined phase could also be added as an example, like `solid`.

Reply:
done